djlink

```diff
@@ -1,11 +1,33 @@
 #!/usr/bin/env bash
 
+OS=`uname -s`
+if test "$OS" == "Darwin" && command -v brew &>/dev/null; then
+  # add keg-only llvm to PATH so llvm-objcopy and llvm-readobj can be found
+  PATH=$PATH:$(brew --prefix llvm)/bin
+fi
+
 if [ -z "$OBJCOPY" ]; then
   OBJCOPY=objcopy
+  if ! $OBJCOPY --version &> /dev/null; then
+     OBJCOPY=llvm-objcopy
+     if ! $OBJCOPY --version &> /dev/null; then
+       echo "Neither objcopy nor llvm-objcopy are installed"
+       exit 1
+     fi
+  fi
 fi
 if [ -z "$STRIP" ]; then
   STRIP=strip
 fi
+if test "$OS" == "Darwin"; then
+  if [ -z "$READOBJ" ]; then
+    READOBJ=llvm-readobj
```

Comment:
There seems to be llvm-readelf
as well, so do you really need this?

Reply:
we can use llvm-readelf instead (it's the same tool with a few minor differences; for Mach-O the output is the same for our use case), it just seemed `llvm-readobj` was more appropriate given that it doesn't parse ELF here but Mach-O.

I could like just call `llvm-readelf` or `llvm-readobj` directly without the env var for MacOS (like it's done with readelf on other OS), then the script will just fail lower down if the tool doesn't exist?

Reply:
done as you suggested.